src/components/editor/InputListLOC.jsx

```diff
@@ -14,9 +14,9 @@ class InputListLOC extends Component {
     this.state = {
       isLoading: false,
       options: [],
-      defaults: []
+      defaults: [],
+      lookupUri: this.props.lookupConfig.uri
```

Comment:
If we're not modifying this value via `this.setState()`, what value do we get out of adding it to component state? Can we use `this.props.lookupConfig.uri` throughout the component instead?

Reply:
Alternatively, if this component only needs the URI and does not need the rest of `lookupConfig`, could the parent component inject just the URI?

Reply:
Yeah, not sure what I was thinking. I changed it to `this.props.lookupConfig.uri`.

Reply:
@jgreben :speech_balloon: 
> Yeah, not sure what I was thinking. 

:laughing: I'm always trying stuff in PRs and then forgetting to back it out. Part of the process!

> I changed it to `this.props.lookupConfig.uri`.

Thank you, sir.